src/plays/airbnb-experiences-clone/styles.css

```diff
@@ -0,0 +1,102 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+nav {
+  height: 70px;
+  display: flex;
+  padding: 20px 36px;
+  box-shadow: 0px 2.98256px 7.4564px rgba(0, 0, 0, 0.1);
+}

```

Comment:
We should avoid overriding the element styles directly and instead use classes.

Reply:
Okay. Will use classes for styling.

Reply:
@Sachin-chaurasiya Done.